drainer/syncer.go

```diff
@@ -422,7 +488,7 @@ ForLoop:
 
 			shouldSkip := false
 
-			if !s.cfg.SyncDDL {
+			if !s.cfg.SyncDDL && !s.loopbackSync.SupportPlugin {
```

Comment:
seems that plugin should not conflict with `SyncDDL`

Reply:
Thanks, I had fixed it.